apps/nextjs-website/src/components/HowTo.tsx

```diff
@@ -25,7 +27,7 @@ const makeHowToProps = ({
     ...(step.icon && {
       stepIcon: {
         ...(isValidMuiIcon(step.icon) && {
-          icon: <Icon>{formatValidMuiIcon(step.icon)}</Icon>,
+          icon: step.icon as keyof typeof MuiIcons, // Temporary (see at the top of the file)
```

Comment:
I would add a task to the backlog and include the link on the ticket to the line in this PR. Additionally, on this PR, I would include the link to the task, and in the code, I would write the task id